docs/source/examples/checks/performance/simple_model_comparison.ipynb

```diff
@@ -67,7 +67,7 @@
     "\n",
     "We will run the check with the **tree** model type. The check will use the default metric defined in deepchecks for classification. This can be overridden by providing an alternative scorer using the `alternative_scorers` parameter.\n",
     "\n",
-    "Note the we are demonstrating on a classification task, but the check also works for regression tasks. For classificaion we will see the metrics per class, and for regression we'll have a single result per metric."
+    "Note the we are demonstrating on a classification task, but the check also works for regression tasks. For classification we will see the metrics per class, and for regression we'll have a single result per metric."
```

Comment:
```suggestion
    "Note that we are demonstrating on a classification task, but the check also works for regression tasks. For classification we will see the metrics per class, and for regression we'll have a single result per metric."
```

Reply:
didn't let me commit this suggestion :-|

Reply:
fixed